modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/representative_form_upload_controller.rb

```diff
@@ -19,20 +19,68 @@ def submit
 
       def upload_scanned_form
         authorize(nil, policy_class: RepresentativeFormUploadPolicy)
-        attachment = PersistentAttachments::VAForm.new
-        attachment.form_id = params['form_id']
-        attachment.file = params['file']
-        raise Common::Exceptions::ValidationErrors, attachment unless attachment.valid?
+        handle_file_upload(PersistentAttachments::VAForm)
+      end
+
+      def upload_supporting_documents
+        authorize(nil, policy_class: RepresentativeFormUploadPolicy)
+        handle_file_upload(PersistentAttachments::VAFormDocumentation)
+      end
+
+      private
+
+      def handle_file_upload(form_type)
+        attachment = create_form(form_type)
```

Comment:
Should we not use `PersistentAttachment#to_pdf` similar to the [example](https://github.com/department-of-veterans-affairs/vets-api/blob/e55baae33f94eae0fdf6cee22f9913e2b0856525/modules/accredited_representative_portal/app/services/accredited_representative_portal/saved_claim_service/attach.rb#L58) in that POC `Attach` class? This code exactly mirrors [how we get a handle of the file when we validate and submit it to Benefits Intake API upon submission](https://github.com/department-of-veterans-affairs/vets-api/blob/e55baae33f94eae0fdf6cee22f9913e2b0856525/app/sidekiq/lighthouse/submit_benefits_intake_claim.rb#L85).